props/saber_fett263_buttons.h

```diff
@@ -381,160 +1813,2794 @@ SaberFett263Buttons() : PropBase() {}
+            track_player_.Free();
+          }
+          track_mode_ = PLAYBACK_OFF;
+          if (!SFX_medit) {
+              talkie.Say(talkie_error_in_15, 15);
+              talkie.Say(talkie_font_directory_15, 15);
+              STDOUT.println("Edit Mode prompts missing");
+              return true;
+          }
+          STDOUT.println("Enter Edit Mode");
+          GenerateIniFiles();
+          PlayMenuSound("medit.wav");
+          current_menu_angle_ = fusor.angle2();
+          menu_top_pos_ = 0;
+          menu_ = true;
+          menu_type_ = MENU_TOP;
+          FastOn();
+        } 
+#else
+        GenerateIniFiles();
+        PlayMenuSound("msave.wav");
+#endif
+        return true;
+
+      case EVENTID(BUTTON_NONE, EVENT_TWIST_RIGHT, MODE_OFF | BUTTON_POWER):
+        if (menu_ && menu_type_ == TRACK_PLAYER) {
+          track_mode_ = PLAYBACK_ROTATE;
+          PlayMenuSound("mrotate.wav");
+          return true;
+        }
+        return false;
+
+      case EVENTID(BUTTON_NONE, EVENT_TWIST_LEFT, MODE_OFF | BUTTON_POWER):
+        if (menu_ && menu_type_ == TRACK_PLAYER) {
+          track_mode_ = PLAYBACK_LOOP;
+          PlayMenuSound("mloop.wav");
+          return true;
+        }
+        return false;
+
+      case EVENTID(BUTTON_AUX, EVENT_HELD_LONG, MODE_ON):
+        if (menu_) {
+          PlayMenuSound("mexit.wav");
+          menu_ = false;
+          menu_type_ = MENU_TOP;
+          return true;
+        }
+        if (rehearse_) {
+          PlayMenuSound("mcancel.wav");
+          rehearse_ = false;
+          return true;
+        }
+        if (!battle_mode_) {
+          battle_mode_ = true;
+          if (SFX_bmbegin) {
+            hybrid_font.PlayCommon(&SFX_bmbegin);
+          } else {
+            hybrid_font.DoEffect(EFFECT_FORCE, 0);
+          }
+        } else {
+          battle_mode_ = false;
+          if (SFX_bmend) {
+            hybrid_font.PlayCommon(&SFX_bmend);
+          } else {
+            beeper.Beep(0.5, 3000);
+          }
+        }
+        return true;
+
+      case EVENTID(BUTTON_AUX, EVENT_HELD_LONG, MODE_ON | BUTTON_POWER):
+        if (!menu_ && saved_gesture_control.powerlock || choreo_) {
+          wav_player.Free();
+          choreo_ = false;
+          battle_mode_ = false;
+          Off();
+        }
+        return true;
+
+
+      case EVENTID(BUTTON_NONE, EVENT_TWIST_RIGHT, MODE_ON):
+        if (wav_player->isPlaying()) return true;
+        if (color_mode_ == CC_COLOR_LIST) {
+          dial_ = (dial_ + 1) % NELEM(color_list_);
+          ONCEPERBLADE(CC_NEW_COLOR)
+          current_preset_.Save();
+          hybrid_font.PlayCommon(&SFX_ccchange);
+          UpdateStyle(current_preset_.preset_num);   
+          return true;
+        }
+        if (!menu_) return false;
+        switch (menu_type_) {
+          case MENU_PRESET:
+            next_preset_fast();
+            if (SFX_quote) {
+              SFX_quote.Select(-1);
```

Comment:
Going to a different font should already undo all selections.
If it doesn't, that's a bug and we should fix that.

Reply:
removed